tests/specs/api.e2e.js

```diff
@@ -1,10 +1,40 @@
 import { notIE } from '../../tools/browser-matcher/common-matchers.mjs'
+import { apiMethods, asyncApiMethods } from '../../src/loaders/api/api-methods'
 
 describe('newrelic api', () => {
   afterEach(async () => {
     await browser.destroyAgentSession()
   })
 
+  it('should expose api methods', async () => {
+    await browser.url(await browser.testHandle.assetURL('api/local-storage-disallowed.html')) // Setup expects before loading the page
+      .then(() => browser.waitForAgentLoad())
+
+    const globalApiMethods = await browser.execute(function () {
+      return Object.keys(window.newrelic)
+    })
+    const agentInstanceApiMethods = await browser.execute(function () {
+      function getAllPropertyNames (obj) {
+        let result = new Set()
+        while (obj) {
+          Object.getOwnPropertyNames(obj).forEach(p => result.add(p))
+          obj = Object.getPrototypeOf(obj)
+        }
+        return [...result]
+      }
+      return getAllPropertyNames(Object.values(newrelic.initializedAgents)[0])
```

Comment:
Would be less dependent on knowing the inheritance, and if that ever change, to just do an `expect` loop through `typeof newrelic.initializedAgents[0][method] === 'function'`, no?

Reply:
Could do that but the code grabs all properties of the object traversing the inheritance chain. Either would work IMHO.